dashboard/app/helpers/ai_lesson_summaries_helper.rb

```diff
@@ -4,7 +4,7 @@ module AiLessonSummariesHelper
   API_KEY = CDO.openai_lesson_summaries_api_key
   MODEL = SharedConstants::EVALUATE_STUDENT_LEARNING_MODEL_VERSION
 
-  def self.get_ai_lesson_summary(lesson_id, user_id = nil, response_format = AiSystemPrompts::LessonSummariesSystemPromptHelper::RESPONSE_FORMATS[:BRIEF_SUMMARY])
```

Comment:
Made this method title a bit more specific since we have lots of methods on "getting", "retrieving", and "requesting" lesson summaries and it was a bit confusing